### DISCUSSION TITLE
Error accessing a cell by name

### DISCUSSION THREAD

**Discussion by ateufel** [exceljs/exceljs#2526] [Ideas & Proposals]
I am using an existing Excel file (with workbook.xlsx.readFile), and it is working perfectly fine when i access a cell by `worksheet.getCell('C4')` (for example). The same cell is named "testcell", but if i try to access it by `worksheet.getCell('testcell')`, i get the following error:

`TypeError: Cannot read property '0' of null`

Is it even possible to access a cell by name? The name is definitely set and correct, i get it when i try this:

`worksheet.getCell('C4').name`

**Comment by mah110020** [comment]
I do not believe defined names have that functionality. The documentation only specifies defined names being used in formulas and date validation.
